pysat/tests/test_meta.py

```diff
@@ -1222,7 +1336,8 @@ def test_transfer_attributes_to_instrument_strict_names(self):
                                                         strict_names=True)
 
     def test_transfer_attributes_to_instrument_strict_names_false(self):
-        """Test attr transfer with strict_names set to True"""
+        """Test attr transfer with strict_names set to True.
```

Comment:
```suggestion
        """Test attr transfer with strict_names set to False.
```

Reply:
I think I was going to combine these with parametrize, decided against it and forgot to update.